smdebug/core/hook.py

```diff
@@ -437,11 +437,6 @@ def _initialize_writers(self, only_initialize_if_missing=False) -> None:
                     self.logger.info(f"Hook is writing from the hook with pid: {os.getpid()}\n")
                 else:
                     self.first_process = False
-                    self.logger.warn(
```

Comment:
move this log to 439 and then we will be logging only once. 
if self.first_process is None:
  log
self.first_process= False

Reply:
added in the latest commit